### DISCUSSION TITLE
about plot window

### DISCUSSION THREAD

**Discussion by mashiro210** [REditorSupport/vscode-R#861] [Q&A]
vs codeでRを使用する際に、plot windowが独立なところが気に入って使用していますが、久しぶりにmacで使用したところvscode内の新たなタブにplotが表示されます
どのようにしたら以前のようにplotが独立して表示されるようになるのでしょうか？

**Comment by eitsupi** [comment]
@mashiro210 How about writing `options(vsc.plot = FALSE)` to Rprofile as described here?
https://github.com/REditorSupport/vscode-R/wiki/Plot-viewer#native-plot-window
